packages/column-views/src/maps/column-navigation/mapbox/state.ts

```diff
@@ -76,13 +76,19 @@ export function ColumnNavigationProvider({
   //   store.setState({ columns: _columns, selectedColumn });
   // }, [projectID, inProcess, columns, getColumns]);
 
-  const _columns = columns ?? useMacrostratColumns(projectID, inProcess);
```

Comment:
We should have both the `columns` and `columnIDs` options available.

- `columns` allows you to pass in a fully separate set of columns as GeoJSON (this is useful for instance for columns that have been pre-filtered or haven't yet been saved to the server)
- `columnIDs` would allow you to pass in a set of IDs to filter based on.
Alternatively, you could allow `columns` to be either an array of numbers or GeoJSON and change things based on which is passed.